docs/tctl/how-to-remove-a-search-attribute-from-a-cluster-using-tctl.md

```diff
@@ -9,6 +9,10 @@ tags:
   - filtered-lists
 ---
 
+## NOTE
+
+> The Temporal tctl documentation covers version 1.16 of the Temporal CLI.
+
 <!-- prettier-ignore -->
 import * as WhatIsASearchAttribute from '../concepts/what-is-a-search-attribute.md'
```

Comment:
We have decided not to use the use of the preview component - so this can be removed and the `<preview>` link can be converted back to Markdown below.